Tests/BackendIntegrationTests/OtherIntegrationTests.swift

```diff
@@ -31,42 +31,6 @@ class OtherIntegrationTests: BaseBackendIntegrationTests {
         expect(info.isComputedOffline) == false
     }
 
-    func testGetCustomerInfoMultipleTimesInParallel() async throws {
```

Comment:
I can't figure out a way to write this test in a way that can't suffer from race conditions.

Reply:
Luckily this is already covered by unit tests.